src/quest_manager/models.py

```diff
@@ -30,6 +30,12 @@ class Category(IsAPrereqMixin, models.Model):
         help_text="Quests that are a part of an inactive campaign won't appear on quest maps and won't be available to students."
     )
 
+    import_id = models.UUIDField(
+        default=uuid.uuid4,
+        unique=True,
+        help_text="Used to link this object across schemas. Don't edit manually unless necessary."
```

Comment:
This help text should be more informative.  Users don't know what `schemas` are.   Let's change to this:

"This value links your campaign to the corresponding campaign within the Library.  Only change this value if you want to disconnect your campaign from the Library."